recipes/extras.rb

```diff
@@ -24,6 +24,9 @@
 
   insert_into_file('bin/setup', Net::HTTP.get(setup_file), after: /^ *chdir APP_ROOT do.*\n/, force: false)
   insert_into_file('bin/update', Net::HTTP.get(update_file), after: /^ *chdir APP_ROOT do.*\n/, force: false)
+
+  gsub_file '.gitignore', /.*\/config\/database.yml.*\n/, ''
```

Comment:
@MProG why did you use `gsub_file` function, if you could remove this lines from template?